crates/cirrus-node-primitives/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "cirrus-node-primitives"
```

Comment:
No strong opinion on this, just want to say `cirrus-node-primitives` is used by both the cirrus and embedded subspace node. If we eventually merge the cirrus binary into the subspace node, not sure we still want the cirrus naming.

Reply:
It is not used by `subspace-node`, at least not yet (in this PR)